### DISCUSSION TITLE
An ambiguity concerning "multiple detections corresponding one groundtruth object " in the "readme.md"

### DISCUSSION THREAD

**Discussion by tendfang** [rafaelpadilla/Object-Detection-Metrics#145] [General]
In "readme.md", the author wrote "In some images there are more than one detection overlapping a ground truth (Images 2, 3, 4, 5, 6 and 7). For those cases the first detection is considered TP while the others are FP." and cited [Int J Comput Vis (2015) 111:98–136]. However, in the original article, the sentence the author referred follows the sorting operation "The detection objects were sorted in a decreasing order by confidence value". That is to say, if there are multiple detections overlapping a ground-truth object with enough IOU, the one with highest confidence value is TP, instead of the so-called "first detection". Or, if the author can clearly mention the sorting operation before that, there should be no ambiguity. What do you think of it?

**Comment by huytranvan2010** [comment]
@tendfang  I confused this problem too. You can look at image 2. Predicted bounding boxes D and E overlap with one ground-truth box. Box E has IoU with the ground-truth box > IoU of the box D, so predicted box E is considered TP, and the predicted box D is considered FP as in the table. **This means the predicted box with the highest IoU will become TP, not the highest confidences score.** Here we assumed that  2 predicted boxes meet the confidence score condition ( > confidence score threshold).

**Comment by rafaelpadilla** [comment]
@tendfang @huytranvan2010 

Thank you for opening this discussion.

You are right. The text sounds confusing and conflicts with the presented example.

@huytranvan2010 got the point precisely: "the predicted box with the highest IoU will become TP, not the highest confidences score."

Will add that into the Read me file.
